### PR TITLE
[SPARK-45779][INFRA][FOLLOWUP] Add more spaces to the prompts

### PR DESCRIPTION
**Pull request by dongjoon-hyun**
### What changes were proposed in this pull request?

This PR aims to add more spaces to the prompts to be consistent with the existing behaviors.

### Why are the changes needed?

Previously, we have these spaces.

### Does this PR introduce _any_ user-facing change?

No.

### How was this patch tested?

Manual review.

### Was this patch authored or co-authored using generative AI tooling?

No.

### CODE REVIEW

File: dev/merge_spark_pr.py [hunk 1]
```diff
@@ -262,7 +262,7 @@ def print_jira_issue_summary(issue):
 
 
 def get_jira_issue(prompt, default_jira_id=""):
-    jira_id = input("%s[%s]: " % (prompt, default_jira_id))
+    jira_id = input("%s [%s]: " % (prompt, default_jira_id))
```
Comment (dongjoon-hyun) [inline dev/merge_spark_pr.py:265]: Our previous prompt has a space like `jira_id = input("Enter a JIRA id [%s]: " % default_jira_id)`.

File: dev/merge_spark_pr.py [hunk 2]
```diff
@@ -275,7 +275,7 @@ def get_jira_issue(prompt, default_jira_id=""):
         if status == "Resolved" or status == "Closed":
             print("JIRA issue %s already has status '%s'" % (jira_id, status))
             return None
-        if input("Check if the JIRA information is as expected(y/n): ").lower() != "n":
+        if input("Check if the JIRA information is as expected (y/n): ").lower() != "n":
```
Comment (dongjoon-hyun) [inline dev/merge_spark_pr.py:278]: Previously, all `(y/n)` has a space separator.

**Comment by dongjoon-hyun** [comment]
Hi, @yaooqinn . While merging, I realized that I missed two spaces. Could you review this?

**Review by MaxGekk — APPROVED** [review]

**Review by yaooqinn — APPROVED** [review]
LGTM

**Comment by dongjoon-hyun** [comment]
Thank you, @MaxGekk and @yaooqinn . Merged to master.

**Review by yaooqinn — APPROVED** [review]
LGTM

**Pull request closed without merging**